### ISSUE TITLE
[A11Y] Unable to exit ProfilePictureActivity from screen-reader

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#3900]
[A11Y] Unable to exit ProfilePictureActivity from screen-reader

Open question: In my opinion we actually do not need this activity for screen-reader.

https://user-images.githubusercontent.com/9396084/136245763-cf4cd455-3a9e-4af9-a5c8-c9a1ebb06d85.mp4

**Comment by rt4914** [comment]
Add a toolbar with `Profile Photo` title.

**Comment by bkaur-bkj** [comment]
@rt4914 I would like to work on this issue
